perception/image_projection_based_fusion/src/fusion_node.cpp

```diff
@@ -53,87 +56,67 @@ FusionNode<Msg, ObjType>::FusionNode(
     rois_number_ = 8;
   }
 
-  // subscribers
-  sub_.subscribe(this, "input", rclcpp::QoS(1).get_rmw_qos_profile());
+  // Set parameters
+  match_threshold_ms_ = static_cast<int>(declare_parameter("match_threshold_ms", 50));
+  timeout_ms_ = static_cast<double>(declare_parameter("timeout_ms", 0.1));
```

Comment:
```suggestion
  match_threshold_ms_ = declare_parameter<double>("match_threshold_ms");
  timeout_ms_ = declare_parameter<double>("timeout_ms");
```
same as <https://github.com/autowarefoundation/autoware.universe/pull/2175/files#r1015132318>

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2175/commits/7807a448f3c772d49ebe50bc6f5fe3863ead6a64

Reply:
Cloud you check again?